.github/workflows/scripts/comments.mjs

```diff
@@ -1,14 +1,70 @@
+import { readFile } from 'node:fs/promises'
+
+/**
+ * Posts the content of a diff as a comment on a Github issue
+ *
+ * @param {GithubActionContext} githubActionContext
+ * @param {number} issueNumber
+ * @param {DiffComment} diffComment
+ */
+export async function commentDiff (
+  githubActionContext,
+  issueNumber,
+  { path, title, markerText }
+) {
+  // Read diff from previous step
+  const diffText = await readFile(path, 'utf8')
```

Comment:
Can we save a whole function by defaulting here?

```suggestion
  const diffText = (await readFile(path, 'utf8')) ?? 'The diff is empty.'
```

Reply:
I like that first one, hadn't thought I could solve it at that point, that'll simplify things greatly! 🙌🏻